### PR TITLE
Debian: install Lunatik on kernel upgrade

### LINKED ISSUE #195: Script to automatically install Lunatik on Debian update
[Here is](https://gist.github.com/jperon/46f17707425b87ec6ed1dc56724e3cd8) a script that, in `/etc/kernel/postinst.d/`, installs Lunatik for the new kernel. Would it be worth a PR? If so, where should I put the script?

### PR DESCRIPTION
**Pull request by jperon**
Fix #195

### CODE REVIEW

**Review by lneto — APPROVED** [review]

File: tools/debian_kernel_postinst_lunatik.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```
Comment (lneto) [inline tools/debian_kernel_postinst_lunatik.sh:1]: Can you please add license here? Dual MIT/GPL2 is preferable.. thanks!

Comment (jperon) [reply tools/debian_kernel_postinst_lunatik.sh:1]: Just done, with copyright assignment so that you may change it at your will.

**Comment by jperon** [comment]
Just added a small change to use all detected cpu cores, and not previously hardcoded `4`.

File: tools/debian_kernel_postinst_lunatik.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# SPDX-FileCopyrightText: (c) 2024 Ring Zero Desenvolvimento de Software LTDA
```
Comment (lneto) [inline tools/debian_kernel_postinst_lunatik.sh:2]: I think you should use your own copyright ;-).. btw, can you also squash your commits? It helps to sanitize the history.. thanks!

**Comment by jperon** [comment]
Here you are!

**Pull request merged**